src/main/java/com/amihaiemil/versioneye/RtRepositories.java

```diff
@@ -43,6 +43,9 @@
  * @author Sherif Waly (sherifwaly95@gmail.com)
  * @version $Id$
  * @since 1.0.0
+ * @todo #112:30min/DEV Modify repository(repositoryKey) method to return
```

Comment:
@amihaiemil The data for Repository can be accessed by json.getString("repo") and there are extra data. So I think it can't be used as we must know whether to access a field directly or get it from "repo" first